crates/dbsp/src/operator/dynamic/aggregate/min.rs

```diff
@@ -68,3 +68,74 @@ where
         accumulator
     }
 }
+
+/// An aggregator that returns the smallest Some() value with non-zero weight
```

Comment:
this looks fine, but perhaps I should push a commit which is using it, and then we should merge

Reply:
good plan!